pom.xml

```diff
@@ -461,6 +462,11 @@
         <artifactId>log4j-over-slf4j</artifactId>
         <version>${slf4j.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.mongodb</groupId>
+        <artifactId>mongodb-driver-sync</artifactId>
```

Comment:
Why the synchronous driver instead of the asynchronous one?

Reply:
The async driver is called mongodb-driver-reactivestreams. I need to investigate the benefits of each, unless you have a particular view?

Reply:
There appears to be little difference - just as long as this is a conscious decision and there is no reason to use async.

Reply:
At the moment I haven't found use of Subscriber pattern in Nessie that would warrant using mongodb-driver-reactivestreams. I might have missed it. So using the sync driver would fit the bill.